syft/frameworks/crypten/__init__.py

```diff
@@ -1,4 +1,46 @@
+import torch
+import syft
 from syft.frameworks.crypten.context import toy_func, run_party
+import crypten.communicator as comm
+import crypten
 
 
-__all__ = ["toy_func", "run_party"]
+def load(tag: str, src: int, id_worker: int):
+    if comm.get().get_rank() == src:
+        worker = syft.local_worker.get_worker(id_worker)
+        result = worker.search(tag)[0].get()
+
+        # file contains torch.tensor
+        if torch.is_tensor(result):
+            # Broadcast load type
+            load_type = torch.tensor(0, dtype=torch.long)
+            comm.get().broadcast(load_type, src=src)
+
+            # Broadcast size to other parties.
+            dim = torch.tensor(result.dim(), dtype=torch.long)
+            size = torch.tensor(result.size(), dtype=torch.long)
```

Comment:
I'm curious why you had to provide the dim + the size: is this because the receving part:
```
# Receive size from source party
            dim = torch.empty(size=(), dtype=torch.long)
            comm.get().broadcast(dim, src=src)
```
should have a fixed size in advance?

Reply:
That's because only the src party knows the dimensions and size of the tensor to be encrypted, and the shares should be created accordingly, this this information should be sent to all the other parties via brodcast.